index.html

```diff
@@ -0,0 +1,110 @@
+    <meta name="viewport" content="width=device-width" />
+    <title>Mobile</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="header">
+      <a class="name" href="assets/images/Vahan.jpg"
+        ><p>Vahan Khachvankian</p></a
+      >
+      <div class="burger">
+        <a href="#"><img src="assets/images/lines-56763.png" alt="logo" /></a>
+      </div>
+    </div>
+
+    <div class="main">
+      <div class="title">
+        <h1>
+          Hey There, <br />
+          I'm Vahan
+        </h1>
+      </div>
+      <div class="head">
+        <p class="pretext">I'am a Software Developer</p>
+      </div>
+      <div class="content">
+        <p class="text">
+          I can help you build a product, feature or website. Look through of
+          some my work and experience! If you like what you see and have a
+          project you need coded, don't hesitate to contact me.
+        </p>
+      </div>
+
+      <nav>
+        <ul id="menu">
+          <li>
+            <a
+              href="https://github.com"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <img
+                class="github"
+                src="assets/images/github.svg"
+                alt="github logo"
+            /></a>
+          </li>
+          <li class="bg">
+            <a
+              href="https://www.vectorstock.com/"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <img
+                class="vector-one"
+                src="assets/images/Vector.png"
+                alt="vector-one" />
+              <img
+                class="vector-two"
+                src="assets/images/VectorN.png"
+                alt="vector-two"
+            /></a>
+          </li>
+          <li>
+            <a
+              href="https://angel.co/"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <img
+                class="angel"
+                src="assets/images/angellist.svg"
+                alt="angel logo"
+            /></a>
+          </li>
+          <li>
+            <a
+              href="https://twitter.com/"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <img
+                class="twitter"
+                src="assets/images/twitter.svg"
+                alt="twitter logo"
+            /></a>
+          </li>
+          <li>
+            <a
+              href="https://medium.com/"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <img
+                class="medium"
+                src="assets/images/medium.svg"
+                alt="medium logo"
+            /></a>
+          </li>
+        </ul>
+        <div class="background">
+          <img class="bottom" src="assets/images/VectorZet.png" alt="bottom" />
+        </div>
```

Comment:
- [ ] Consider using this image as a background image instead of an `img` tag, this is to have a separation of concerns which improves your code structure. This image is for decoration and therefore required to be a background image in the CSS.
- [ ] Also try making the image to match that of your chosen template 😄 

Your design:
<img width="104" alt="chrome_vYi0yrynPH" src="https://user-images.githubusercontent.com/81468675/143865692-fef61401-1f2d-485e-90d2-4fef1c80c1c9.png">

Figma design:
<img width="54" alt="chrome_5dhg8ygflE" src="https://user-images.githubusercontent.com/81468675/143865753-e1bf4598-eb45-4737-af91-3ecd97c6fd2c.png">